packages/edit-site/src/components/dataviews/in-filter.js

```diff
@@ -18,12 +19,16 @@ export default ( { filter, view, onChangeView } ) => {
 			? ''
 			: valueFound.value;
 
+	const id = useInstanceId( InFilter, 'dataviews__filters-in' );
```

Comment:
Nice, thanks. Could we use as `id` something like `dataviews__filters-in-author` instead of `dataviews__fisters-in-0`?

We could take the field name from `filter.field` (`author`, `status`) and they are unique. `filter.name` is more of a user-facing name (can have spaces, etc.) so it wouldn't work that well I guess.

Reply:
Should we use React's useId instead of useInstanceId? Feels like the latter should be deprecate these days.

Reply:
Happy to go with whatever works for the team. If we're confident that `filter.field` is unique, and each filter type will only appear once on the page, then makes sense to use that – in which case, we won't need to apply either `useInstanceId` or `useId`.